system-test/datastore.ts

```diff
@@ -619,6 +619,32 @@ describe('Datastore', () => {
         });
     });
 
+    it('should run a datastore query as a stream via query#runStream', done => {
+      const q = datastore.createQuery('Character').hasAncestor(ancestor);
+      let resultsReturned = 0;
+      q.runStream()
+        .on('error', done)
+        .on('data', () => resultsReturned++)
+        .on('end', () => {
+          assert.strictEqual(resultsReturned, characters.length);
+          done();
+        });
+    });
+
+    it('should run a transaction query as a stream via query#runStream', done => {
+      const transaction = datastore.transaction();
+      const q = transaction.createQuery('Character').hasAncestor(ancestor);
+      let resultsReturned = 0;
+      q.runStream()
+        .on('error', done)
+        .on('data', () => resultsReturned++)
+        .on('end', () => {
+          console.log('total results are => ' + resultsReturned);
```

Comment:
I don't think we should log from the tests, just keeping the output clean 👍

Reply:
Done! 
Thanks

Anything else jumps out?

Reply:
Nope, LGTM. Thanks!